pika/adapters/select_connection.py

```diff
@@ -373,6 +389,19 @@ def start(self):
             self.poll()
```

Comment:
@wjps, `KQueuePoller.start()` still references `self.open` in `while self.open:`, which no one sets or even initializes any longer. Should this be `while not self.stopping:` instead, like in the other pollers?